website/src/categories/components/tree-view/tree-view.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tree View
+description: The Tree View organize data into a tree-like structure with collapsible and expandable nodes. It allows users to navigate complex hierarchical information, such as directories and categories. The component typically includes parent and child nodes, where parent nodes can be expanded to reveal their children.
+componentType: feedback & status
+shortDescription:  The Tree View component displays hierarchical data in a structured, interactive, and navigable manner, allowing users to explore nested elements efficiently.
+layout: "categories/components/components.njk"
+tags: component
+packageName: tree-view
+storybookCategory: feedback-status
+eleventyNavigation:
+  parent: Components
+  key: Tree View
+  status: draft
```

Comment:
Please use `planned` here, so it will not be visible in the documentation website yet, but only in the sidebar with `planned` badge.

https://github.com/sl-design-system/components/tree/main/website#how-to-change-component-status-in-the-navigation